TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/main/resources/META-INF/services/org.opensearch.migrations.transform.IJsonTransformerProvider

```diff
@@ -0,0 +1 @@
+org.opensearch.migrations.transform.Es2OsPost23
```

Comment:
I don't like the casing or the two different numbers.  Since the 23 is used to represent a sequence number, we should probably not ALSO load a number as an abbreviation.  That leaves something like EsToOsPost23.  Then we have the casing.  I didn't want to use underscores before, but they seem like the least bad option.... What do you think of ES_OS_Post23?  We'll likely have lots of occurrences of ES/OS (& other things too), so coming up w/ abbreviations and establishing underscores could allow this to scale.